contrib/dunst_xr_theme_changer.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env sh
```

Comment:
```
$ sh ./dunst_xr_theme_changer.sh
./dunst_xr_theme_changer.sh: 56: ./dunst_xr_theme_changer.sh: mapfile: not found
./dunst_xr_theme_changer.sh: 63: ./dunst_xr_theme_changer.sh: Syntax error: "(" unexpected
```

It should probably use `bash` rather than `sh`.